RedHat-UX/red-hat-design-system
# docs: build status tables from component docs

## What I did

1. copy the data from https://drive.google.com/u/0/open?id=1RINxMTusHWQ7m7Dljis96c6D44CHgtHprQ7mUhWd-Hc into component docs pages
2. remove the rhds-component-status webcomponent
3. add a `componentStatus` shortcode


## Testing Instructions

1. check the deploy preview, ensure status tables on component pages look correct

## Notes to Reviewers
I'd like to review the categories here, i'm not sure how accurately the table headings (PFE repo, adobe target, etc) reflect our plans anymore.
